### ISSUE TITLE
iTerm2 3.0.0 IndexError

### ISSUE THREAD

**Issue by kaushalmall** [wouterdebie/i2cssh#51]
Updated to iTerm 2 version 3.0.0 last night 

![image](https://cloud.githubusercontent.com/assets/4323558/15787869/7928e9ee-2979-11e6-8642-5f6b98503d91.png)

Since then i2cssh is throwing the error below when I run the command 

`i2cssh -m username@server1,username@server2 -Xi=/path/to/private/key`

```
/Library/Ruby/Gems/2.0.0/gems/rb-scpt-1.0.1/lib/rb-scpt.rb:177:in `block in pack': Unknown Keyword: :terminal (IndexError)
    from /Library/Ruby/Gems/2.0.0/gems/rb-scpt-1.0.1/lib/rb-scpt.rb:177:in `fetch'
    from /Library/Ruby/Gems/2.0.0/gems/rb-scpt-1.0.1/lib/rb-scpt.rb:177:in `pack'
    from /Library/Ruby/Gems/2.0.0/gems/rb-scpt-1.0.1/lib/_aem/send.rb:37:in `block in initialize'
    from /Library/Ruby/Gems/2.0.0/gems/rb-scpt-1.0.1/lib/_aem/send.rb:37:in `each'
    from /Library/Ruby/Gems/2.0.0/gems/rb-scpt-1.0.1/lib/_aem/send.rb:37:in `initialize'
    from /Library/Ruby/Gems/2.0.0/gems/rb-scpt-1.0.1/lib/aem.rb:218:in `new'
    from /Library/Ruby/Gems/2.0.0/gems/rb-scpt-1.0.1/lib/aem.rb:218:in `event'
    from /Library/Ruby/Gems/2.0.0/gems/rb-scpt-1.0.1/lib/rb-scpt.rb:506:in `_send_command'
    from /Library/Ruby/Gems/2.0.0/gems/rb-scpt-1.0.1/lib/rb-scpt.rb:642:in `method_missing'
    from /Library/Ruby/Gems/2.0.0/gems/i2cssh-1.16.0/lib/i2cssh.rb:19:in `initialize'
    from /Library/Ruby/Gems/2.0.0/gems/i2cssh-1.16.0/bin/i2cssh:298:in `new'
    from /Library/Ruby/Gems/2.0.0/gems/i2cssh-1.16.0/bin/i2cssh:298:in `<top (required)>'
    from /usr/local/bin/i2cssh:23:in `load'
    from /usr/local/bin/i2cssh:23:in `<main>
```

**Comment by bcantoni** [comment]
@kaushalmall it looks like you might be on an older version of i2cssh (1.16.0)? If so you can upgrade to the latest version (2.0.0) which includes support for iTerm2 3.0.

**Comment by kaushalmall** [comment]
@bcantoni apologize if this is a stupid question, but is the upgrade an uninstall and install of the new version or is there a better way?

**Comment by bcantoni** [comment]
I believe you can just do `gem update i2cssh` and it should install the latest release. If not, try `gem uninstall i2cssh` followed by `gem install i2cssh`.

**Comment by dellis23** [comment]
Updating fixed the issue for me.

**Comment by chrisagiddings** [comment]
@bcantoni I'm thinking the `README.md` should be updated so anyone new to `i2cssh ` doesn't install an old version and immediately have to update when they get errors.

If the `v1.16.0` is required for older macOS releases or iTerm(2) releases, this should also be noted in the `README.md` file.

Being new myself this transition helped solve the issue on a Touchbar macBook Pro running High Sierra. I followed the `README.md` instructions and received the noted error. After updating to `v2.2.0` this specific issue went away, but I found another problem afterward.
